mypy/util.py

```diff
@@ -317,7 +317,7 @@ def write_junit_xml(
     # checks for a directory structure in path and creates folders if needed
     xml_dirs = os.path.dirname(os.path.abspath(path))
     if not os.path.isdir(xml_dirs):
```

Comment:
Let's remove this condition as it's now redundant.

Reply:
True, did remove it.